isobus/include/isobus/isobus/can_parameter_group_builder.hpp

```diff
@@ -0,0 +1,503 @@
+#pragma once
+
+#include <vector>
+
+namespace isobus
+{
+	// Note that this class currently only works for packets eight bytes or fewer, because that is
```

Comment:
Ideally we'd want it to work up to 117440505 bytes to meet the generic use case.

Reply:
Ah, that's an old comment I forgot to remove.  It did only do eight bytes, but I removed that restriction already.